public/js/controllers/game.js

```diff
@@ -103,6 +103,7 @@ angular.module('mean.system')
 
     $scope.winningColor = function($index) {
       if (game.winningCardPlayer !== -1 && $index === game.winningCard) {
+        //console.log('this is from the controllers>>>', game)
```

Comment:
If the comment is not useful to understanding the code, you might want to remove it.

Reply:
Thank you. I will.